src/components/LikeThought.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import './Thoughts.css'
+import { GetThoughts } from './Thoughts'
+
+export const LikeThoughts = ({ index }) => {
+	const [likes, setLikes] = useState(index.hearts)
+
+const handleLike = (index) => {
```

Comment:
Similarly, the parameter for handleLike is named index, which could be renamed to reflect that it's a thought object :)